tests/test_store.py

```diff
@@ -214,6 +220,34 @@ def test_view_query(
         # tests that the query was valid and successful, that a view subset has actually been retrived
         assert synapse_store_special_scope.storageFileviewTable.empty is False
 
+    @pytest.mark.parametrize(
+        "asset_view,columns,expectation",
+        [
+            ("syn62339865", ["path"], pytest.raises(ValueError)),
+            ("syn62340177", ["id"], pytest.raises(ValueError)),
+            ("syn23643253", ["id", "path"], does_not_raise()),
+        ],
+    )
+    def test_view_query_exception(
```

Comment:
In addition, for `test_view_query`, I am seeing the logic of "only perform the query if it is different from the previous query" not being tested. Can that also be added to this PR? If not, can an additional ticket be created?

Reply:
Thanks for catching! @linglp